src/julia-syntax.scm

```diff
@@ -2424,6 +2424,10 @@
                                        (cdr x)
                                        (list x)))
                                  a)))
+                  (if (any (lambda (x)
+                             (and (pair? x) (pair? (car x)) (eq? (caar x) '...)))
```

Comment:
Doesn't this only check the first item in each row? `(pair? (car x)) (eq? (caar x) '...))` could also be replaced by `(vararg? (car x))`.

Reply:
Of course, you're right. Wasn't aware of the vararg function, I'll try it.